presentation/src/main/java/com/stop/TimeConverter.kt

```diff
@@ -0,0 +1,26 @@
+package com.stop
+
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+fun makeFullTime(time: String): Calendar {
+    val currentTime = System.currentTimeMillis()
+    val currentFormat = SimpleDateFormat("yyyy:MM:dd", Locale.getDefault())
+    val currentDateTime = currentFormat.format(currentTime)
+    val fullTime = "$currentDateTime:$time"
+
+
+    val dateFormat = SimpleDateFormat("yyyy:MM:dd:HH:mm:ss", Locale.getDefault())
+    var dateTime = Date()
+    try {
+        dateTime = dateFormat.parse(fullTime) as Date
+    } catch (e: ParseException) {
+        e.printStackTrace()
+    }
```

Comment:
```suggestion
    val dateTime = try {
        dateFormat.parse(fullTime) as Date
    } catch (e: ParseException) {
        e.printStackTrace()
        Date()
    }
```
try catch body에서 선언한 변수를 body 밖에서 사용하지 못해서 이렇게 선언한 것 같은데 맞을까요?
위와 같이 변경하면 변수를 사전에 선언하지 않아도 됩니다.

Reply:
c635655352f35cba72c4061e63d43b66585d0b04